tasks/install_powershell.json

```diff
@@ -42,6 +42,14 @@
       "description": "The number of retries in case of network connectivity failures",
       "type": "Optional[Integer]",
       "default": 5
+    },
+    "username": {
+      "description": "The username to use when downloading from a source location requiring authentication",
+      "type": "Optional[String]"
+    },
+    "password": {
+      "description": "The password to use when downloading from a source location requiring authentication",
+      "type": "Optional[String]"
```

Comment:
Same here, are empty strings allowed, or should it be updated to `Optional[String[1]]`? And can we use Sensitive as well for the password?

Reply:
Moved discussion to Windows PR